.eslintrc.js

```diff
@@ -56,6 +56,7 @@ module.exports = {
   env: {
     jasmine: true,
     node: true,
+    es2020: true,
```

Comment:
This was initially added for the BigInt changes in the postgres model, however it's no longer needed.     I think we should probably *want* this, but willing to remove if strong feelings exist that we *shouldn't* allow this.